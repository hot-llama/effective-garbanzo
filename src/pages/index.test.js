import React from 'react'
import { shallow } from 'enzyme'

import Layout from '../components/layout'
import IndexPage from '.'

describe('Index Page', () => {
  it('renders a <Layout /> component', () => {
    const wrapper = shallow(<IndexPage />)

    expect(wrapper.find(Layout)).toMatchSnapshot()
  })
})
