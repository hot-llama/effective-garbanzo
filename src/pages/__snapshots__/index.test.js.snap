// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders a <Layout /> component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): ShallowWrapper {
    Symbol(enzyme.__root__): [Circular],
    Symbol(enzyme.__unrendered__): <IndexPage />,
    Symbol(enzyme.__renderer__): Object {
      "batchedUpdates": [Function],
      "getNode": [Function],
      "render": [Function],
      "simulateEvent": [Function],
      "unmount": [Function],
    },
    Symbol(enzyme.__node__): Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1>
            Hi people
          </h1>,
          <p>
            Welcome to your new Gatsby site.
          </p>,
          <p>
            Now go build something great.
          </p>,
          <Link
            to="/app/details"
          >
            Details
          </Link>,
          <p>
            hello wtf
          </p>,
          <Router>
            <PrivateRoute
              component={[Function]}
              path="/app/details"
            />
          </Router>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hi people",
          },
          "ref": null,
          "rendered": "Hi people",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to your new Gatsby site.",
          },
          "ref": null,
          "rendered": "Welcome to your new Gatsby site.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Now go build something great.",
          },
          "ref": null,
          "rendered": "Now go build something great.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Details",
            "to": "/app/details",
          },
          "ref": null,
          "rendered": "Details",
          "type": "Link",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "hello wtf",
          },
          "ref": null,
          "rendered": "hello wtf",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <PrivateRoute
              component={[Function]}
              path="/app/details"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "component": [Function],
              "path": "/app/details",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
    Symbol(enzyme.__nodes__): Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <h1>
              Hi people
            </h1>,
            <p>
              Welcome to your new Gatsby site.
            </p>,
            <p>
              Now go build something great.
            </p>,
            <Link
              to="/app/details"
            >
              Details
            </Link>,
            <p>
              hello wtf
            </p>,
            <Router>
              <PrivateRoute
                component={[Function]}
                path="/app/details"
              />
            </Router>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Hi people",
            },
            "ref": null,
            "rendered": "Hi people",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to your new Gatsby site.",
            },
            "ref": null,
            "rendered": "Welcome to your new Gatsby site.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Now go build something great.",
            },
            "ref": null,
            "rendered": "Now go build something great.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Details",
              "to": "/app/details",
            },
            "ref": null,
            "rendered": "Details",
            "type": "Link",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "hello wtf",
            },
            "ref": null,
            "rendered": "hello wtf",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <PrivateRoute
                component={[Function]}
                path="/app/details"
              />,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "component": [Function],
                "path": "/app/details",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": [Function],
          },
        ],
        "type": [Function],
      },
    ],
    Symbol(enzyme.__options__): Object {
      "adapter": ReactSixteenAdapter {
        "options": Object {
          "enableComponentDidUpdateOnSetState": true,
          "lifecycles": Object {
            "componentDidUpdate": Object {
              "onSetState": true,
            },
            "getDerivedStateFromProps": true,
            "getSnapshotBeforeUpdate": true,
            "setState": Object {
              "skipsComponentDidUpdateOnNullish": true,
            },
          },
        },
      },
      "attachTo": undefined,
      "hydrateIn": undefined,
    },
  },
  Symbol(enzyme.__unrendered__): null,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1>
          Hi people
        </h1>,
        <p>
          Welcome to your new Gatsby site.
        </p>,
        <p>
          Now go build something great.
        </p>,
        <Link
          to="/app/details"
        >
          Details
        </Link>,
        <p>
          hello wtf
        </p>,
        <Router>
          <PrivateRoute
            component={[Function]}
            path="/app/details"
          />
        </Router>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hi people",
        },
        "ref": null,
        "rendered": "Hi people",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to your new Gatsby site.",
        },
        "ref": null,
        "rendered": "Welcome to your new Gatsby site.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Now go build something great.",
        },
        "ref": null,
        "rendered": "Now go build something great.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Details",
          "to": "/app/details",
        },
        "ref": null,
        "rendered": "Details",
        "type": "Link",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "hello wtf",
        },
        "ref": null,
        "rendered": "hello wtf",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <PrivateRoute
            component={[Function]}
            path="/app/details"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "component": [Function],
            "path": "/app/details",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1>
            Hi people
          </h1>,
          <p>
            Welcome to your new Gatsby site.
          </p>,
          <p>
            Now go build something great.
          </p>,
          <Link
            to="/app/details"
          >
            Details
          </Link>,
          <p>
            hello wtf
          </p>,
          <Router>
            <PrivateRoute
              component={[Function]}
              path="/app/details"
            />
          </Router>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hi people",
          },
          "ref": null,
          "rendered": "Hi people",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to your new Gatsby site.",
          },
          "ref": null,
          "rendered": "Welcome to your new Gatsby site.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Now go build something great.",
          },
          "ref": null,
          "rendered": "Now go build something great.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Details",
            "to": "/app/details",
          },
          "ref": null,
          "rendered": "Details",
          "type": "Link",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "hello wtf",
          },
          "ref": null,
          "rendered": "hello wtf",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <PrivateRoute
              component={[Function]}
              path="/app/details"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "component": [Function],
              "path": "/app/details",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
